### ISSUE TITLE
Other publishing options.

### ISSUE THREAD

**Issue by simondlr** [simondlr/theblockchain#10]
There's a lot of publishing options available!

Settling on leanpub for now while I write it (really enjoy it!). If you have any other publishing options I should look at, please add here for discussion.

Current others:
https://www.gitbook.com/

**Comment by richoakley** [comment]
Has anyone tried publishing something like this via Patreon?

**Comment by simondlr** [comment]
@richoakley interesting point. Haven't thought about that. You would do something like "pay-per-chapter"? Or fund someone a monthly "pay" to keep churning out new stuff? What would patron-only posts look like?

I'm sure someone has probably done something similar. Should probably check/read/research. Should be interesting to see examples!

**Comment by richoakley** [comment]
Yeah, would be cool if you could either pay once-off for everything produced so far, or pay-per-chapter to keep adding new stuff? Was just a random thought, but could be a fun experiment!

**Comment by simondlr** [comment]
Definitely something to consider! :)
